server/api.py

```diff
@@ -4,81 +4,78 @@
 import json
 
 api = Blueprint("api", __name__)
-
 CORS(api)
 
+api.secret_key = "ABC"
+
 
-@api.route('/mentor/signup', methods=['POST'])
+@api.route("/mentor/signup", methods=["POST"])
 def mentor_signup():
     """ Adds new user to User table and new mentor to mentor table """
     
     form_data = request.get_json()
     new_user = create_user(form_data, "mentor")
-
     # add new mentor to Mentor table
     new_company = form_data["company"]
     new_knowledge = form_data["knowledge"]
-
     new_mentor = Mentor(user_id=new_user.id, company=new_company, knowledge=new_knowledge)
-
+    
     db.session.add(new_mentor)
     db.session.commit()
-    
-    return json.dumps({'success':True}), 200, {'ContentType':'application/json'}
+  
+    return jsonify({"success":True, "user_id" : new_user.id}), 200, 
         
-@api.route('/mentee/signup', methods=['POST'])
+@api.route("/mentee/signup", methods=["POST"])
 def mentee_signup():
     """ Adds new user to User table and new mentee to mentor table """
 
     form_data = request.get_json()
     new_user = create_user(form_data, "mentee")  
-
     # add mentee to Mentee table
     new_need_help = form_data["need_help"]
     new_mentee = Mentee(user_id=new_user.id, need_help=new_need_help)
     db.session.add(new_mentee)
     db.session.commit()
 
-    return jsonify({'success':True}), 200
+    return jsonify({"success":True, "user_id" : new_user.id}), 200
 
-@api.route("/handle-login", methods=["POST"])
+@api.route("/login", methods=["POST"])
 def handle_login():
     """ Logs in existent User """
 
     form_data = request.get_json()
 
-    email = form_data['email']
+    email = form_data["email"]
     password = form_data["password"]
 
     user = User.query.filter_by(email=email).first()
 
     if password == user.password:
-        session["logged_in_user"] = user
-        return 200
+
+        return jsonify({"success":True, "user_id" : user.id},), 200
     else:
         return "Incorrect email or password" 
 
-@api.route("/matching")
+@api.route("/matching", methods=["POST"])
 def matching():
-    #who's in session? Mentor or Mentee?
-    user_id = session.get('user_id')
```

Comment:
I think this is a more secure way of checking for the current user, what do you think?